addon-test-support/@ember/test-helpers/-internal/build-registry.ts

```diff
@@ -31,9 +31,11 @@ function exposeRegistryMethodsWithoutDeprecations(container: any) {
   for (let i = 0, l = methods.length; i < l; i++) {
     let method = methods[i];
 
-    if (method in container) {
+    if (method && method in container) {
       container[method] = function (...args: unknown[]) {
-        return container._registry[method](...args);
+        // SAFETY: `method` is defined because we *just* checked that it is in
+        // the conditional wrapping this.
```

Comment:
Would this be fixed by changing it to `const method` above?

Reply:
Good question. I'll check on it and see if I can rewrite in a way that makes it clear to TS—it throws away all control flow-based narrowing for objects like that once in methods because the behavior of the closure means it cannot guarantee. But the rewrite you suggested might work!

Reply:
Good catch; it does; I'll update!